domain/src/main/java/nl/tehdreamteam/se42/domain/Message.java

```diff
@@ -11,12 +11,12 @@
  */
 @Entity
 @Table(name = "message")
-public class Message {
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+public abstract class Message<T> {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Document the parameter(s): <T> [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)